### PR TITLE
f-enhancement/gh-403-archive-menu-for-employees

### PR DESCRIPTION
**Pull request by aberonja**
GH issue: #403

### CODE REVIEW

File: src/main/frontend/src/components/employees/Employees.js [hunk 1]
```diff
@@ -14,10 +17,36 @@ const EmployeeContainer = styled.div`
...
   }
 
+  changeEmployeesView = archived => {
+    this.setState(prevState => {
+      return {
+        ...prevState,
+        archived: !prevState.archived
+      };
+    });
+    this.props.resetEmployees();
```
Comment (iscekic) [inline src/main/frontend/src/components/employees/Employees.js]: `\n` 🦆

File: src/main/frontend/src/components/modal/HandleRequestModal.js [hunk 2]
```diff
@@ -117,7 +117,8 @@ class handleRequestModal extends Component {
           </div>
           <div>
             Leave type:
-            <CapitalizedText>{absence.absenceType}</CapitalizedText>
+            <CapitalizedText>{absence.absenceType}</CapitalizedText> -{' '}
```
Comment (iscekic) [inline src/main/frontend/src/components/modal/HandleRequestModal.js]: 🍡

File: src/main/frontend/src/store/reducers/employeesReducer.js [hunk 3]
```diff
@@ -12,6 +12,9 @@ const actionHandlers = {
       last: action.payload.last
     };
   },
+  [resetEmployees]: () => {
```
Comment (iscekic) [inline src/main/frontend/src/store/reducers/employeesReducer.js]: Extract this fn

File: src/main/java/eu/execom/hawaii/repository/UserRepository.java [hunk 4]
```diff
@@ -20,7 +20,7 @@
 
   boolean existsByEmail(String email);
 
-  Page<User> findAllByUserStatusType(List<UserStatusType> userStatusType, Pageable pageable);
+  Page<User> findAllByUserStatusTypeIn(List<UserStatusType> userStatusType, Pageable pageable);
```
Comment (iscekic) [inline src/main/java/eu/execom/hawaii/repository/UserRepository.java]: Should these changes be here?

Comment (aberonja) [reply src/main/java/eu/execom/hawaii/repository/UserRepository.java]: It was there before back-end PR didn't go through, now it shouldn't be here.

**Pull request merged**